codegenerator/cli/src/config_parsing/postgres_types.rs

```diff
@@ -20,11 +20,14 @@ impl Primitive {
             Self::Boolean => "Boolean".to_string(),
             Self::Text => "Text".to_string(),
             Self::Integer => "Integer".to_string(),
-            Self::Numeric => "Numeric".to_string(),
+            Self::Numeric(None) => "Numeric".to_string(),
+            Self::Numeric(Some((precision, scale))) => {
+                format!("Custom(\"NUMERIC({}, {})\")", precision, scale)
+            }
```

Comment:
This is used both for big int and for big decimal and obviously inside the arrays of both of those.